### PR TITLE
Now it populates

### PR DESCRIPTION
**Pull request by zafertaskent**
Added to Data model

### CODE REVIEW

File: api/controllers/DataController.js [hunk 1]
```diff
@@ -27,11 +27,10 @@ module.exports = {
...
     },
     find: function(req, res) {
-        Data.find().populate("department").populate("indicator").populate("criterias").populate("years").exec(function(err, data) {
+        Data.find().populate("department").populate("subdepartment").populate("indicator").populate("criterias").populate("years").exec(function(err, data) {
             res.json(data);
         });
     },
-
     findOne: function(req, res) {
         Data.findOne(req.param('id')).populate("department").populate("indicator").populate("criterias").populate("years").exec(function(err, data) {
```
Comment (adilatilgan) [inline api/controllers/DataController.js]: You should populate it in findOne also.

File: api/models/Data.js [hunk 2]
```diff
@@ -11,6 +11,9 @@ module.exports = {
         department: {
             model: 'Department'
         },
+        subDepartmen:{
```
Comment (adilatilgan) [inline api/models/Data.js]: it seem there is typo here you should test before PR

Comment (zafertaskent) [inline api/models/Data.js]: Sorry I fix it.

File: api/controllers/DataController.js (hunk 1 above)
Comment (zafertaskent) [inline api/controllers/DataController.js]: added

**Comment by adilatilgan** [comment]
I cannot see any sub-department and departmentDesc in my local? @zafertaskent

**Comment by zafertaskent** [comment]
While creating data subDepartment and departmentDesc should be posted.

**Comment by adilatilgan** [comment]
But @berkantesk @ogzhnozlci does not know about it yet. Create Data end point has 2 more fields now. subDepartment and departmentDesc , FE team also post this data while creating. subDepartment must be id of the subDepartment, and departmentDesc is free text area. 

I am merging and deploying this, but it seems like we have no test on this issue.

**Pull request merged**